plugins/chat/config/routes.rb

```diff
@@ -27,6 +27,7 @@
     get "/channels/:channel_id/memberships" => "channels_memberships#index"
     post "/channels/:channel_id/memberships" => "channels_memberships#create"
     delete "/channels/:channel_id/memberships/me" => "channels_current_user_membership#destroy"
+    delete "/channels/:channel_id/memberships/:username" => "channels_memberships#destroy"
```

Comment:
Actually there's a problem with using `username` here, because some usernames, e.g. containing a `.` won't work in a URL, and we also have a magic value `/me`, so if someone takes that username that's another problem.

Changing this to use user ID instead.